tests/data/AB1_design_expected_0_70_r.yaml

```diff
@@ -0,0 +1,11 @@
+!<lsst.verify.Measurement>
+blobs:
+- !<lsst.verify.Blob>
+  data: {}
+  identifier: 9b79b6610f3441188d7395fdc5d50809
+  name: AB1_design
+identifier: d42026f20b544bf49aec380efca3b583
+metric: AB1_design
+notes: {}
+unit: marcsec
+value: 7.965623934179928
```

Comment:
Is it expected that the reference values for r and i band are the same for AB1?

Reply:
This is a good question.  The way AB1 is defined, I'd expect the r-band value to be zero.

Reply:
Thanks. Sounds good to check this on a separate ticket.